docs/service-fabric/refactor-migrated-app.md

```diff
@@ -0,0 +1,155 @@
+---
+title: Refactoring an Azure Service Fabric Application migrated from Azure Cloud Services 
+description: How to refactor an existing Azure Service Fabric Application migrated from Azure Cloud Services
+author: petertay
+ms.date: 01/30/2018
+---
+# Refactoring an Azure Service Fabric Application migrated from Azure Cloud Services
+
+[![GitHub](../_images/github.png) Sample code][sample-code]
+
+This article describes refactoring an existing Azure Service Fabric application to a more granular architecture. This article focuses on the design, packaging, performance, and deployment considerations of the refactored Service Fabric application.
+
+## Scenario
+
+As discussed in the previous article, [migrating an Azure Cloud Services application to Azure Service Fabric][migrate-from-cloud-services], in 2012 the patterns & practices team documented the process for designing and implementing a Cloud Services application in Azure. The book describes a fictitious company named Tailspin that wants to create a Cloud Services application named **Surveys**. The Surveys application allows users to create and publish surveys that can be answered by the public. The following diagram shows the architecture of this version of the Surveys application:
```

Comment:
> The book describes

Not clear which book is meant, since we're also referencing an article (which is different from the book)